include/uv.h

```diff
@@ -409,6 +409,37 @@ struct uv_shutdown_s {
   } u;                                                                        \
   UV_HANDLE_PRIVATE_FIELDS                                                    \
 
+#define UV_TIMER_PRIVATE_FIELDS                                               \
```

Comment:
On some handles we have some fields directly in their struct definition which are marked as private with a comment.  Check if there are repetitions please.

Also, ignore the first part of my message, I think it looks better with the define, for now.